docs/BreakingChanges/103 Right clicking on a WPF DataGrid row header changes the DataGrid selection.md

```diff
@@ -0,0 +1,34 @@
+## 103: Right clicking on a WPF DataGrid row header changes the DataGrid selection
```

Comment:
{
  "Id": "103",
  "Title": "Right clicking on a WPF DataGrid row header changes the DataGrid selection",
  "Details": "Right-clicking a selected DataGrid row header while multiple rows are selected results in the DataGrid's selection changing to only that row.",
  "Suggestion": "This issue has been fixed in the .NET Framework 4.6 and may be addressed by upgrading to that version of the .NET Framework.",
  "Link": "http://connect.microsoft.com/VisualStudio/feedback/details/1001771/right-clicking-on-wpf-data-grid-row-header-changes-selection",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Windows.Controls.DataGrid.#ctor"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.6",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/1064465",
  "Notes": "Source analyzer status: Pri 3, Planned - No tracking work item yet",
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Windows.Controls.DataGrid.#ctor